README.md

```diff
@@ -1,5 +1,7 @@
 # LMC Coding Standard for PHP
 
+[![Latest Stable Version](https://img.shields.io/packagist/v/lmc/coding-standard.svg?style=flat-square)](https://packagist.org/packages/lmc/coding-standard)
```

Comment:
what about other icons? such as `build`, `licence` or `downloads`?

Reply:
Maybe later... But build is imho irrelevant, licence is shown by Github and downloads doesn't make sense to me fort this package. Will se in the future...